### ISSUE TITLE
Feature Request: tf.train.MonitoredTrainingSession implementation for slim.learning.train

### ISSUE THREAD

**Issue by chahna107** [tensorflow/tensorflow#15793]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: N/A
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: N/A
- **TensorFlow installed from (source or binary)**: N/A
- **TensorFlow version (use command below)**: N/A
- **Python version**:  N/A
- **Bazel version (if compiling from source)**: N/A
- **GCC/Compiler version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: N/A
- **Exact command to reproduce**: N/A

This is a feature request:
Issue #6263 states that tf.train.Supervisor is to be deprecated. Since the slim.learning.train uses Supervisor internally, is it still advisable to use it?
Or are there any chances to implement the slim.learning.train to use tf.train.MonitoredSession instead of Supervisor?

@sguada 

Thanks!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by chahna107** [comment]
Updated the fields to N/A.
Thanks!

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
The original poster has replied to this issue after the stat:awaiting response label was applied.

**Comment by reedwm** [comment]
/CC @sguada. Note the Supervisor is created [here](https://github.com/tensorflow/tensorflow/blob/09ec6b2c59d05230b275a14bbc54a8890b6b1c62/tensorflow/contrib/slim/python/slim/learning.py#L721). Should we mark as contributions welcome?

**Comment by sguada** [comment]
Sure
